Makefile

```diff
@@ -121,7 +121,9 @@ install:
 GIT_VERSION: .git
 	echo "$(GIT_VERSION)" >GIT_VERSION
 
-docker-image: clean GIT_VERSION
+docker-image: clean docker-image-no-clean
```

Comment:
This can be made in a different PR. I think @nebril wants this a lot.

Reply:
Done & merged (#6132).